### PR TITLE
Add nyc (Istanbul) -> coveralls coverage

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

**Comment by vogre** [comment]
Screen Shot 2019-12-29 at 16 34 58
Could not drill down into the file on coveralls website, but their design is on point.

**Comment by arielshaqed** [comment]
> Screen Shot 2019-12-29 at 16 34 58
> Could not drill down into the file on coveralls website, but their design is on point.

Good catch, thanks!  Fixed that by pulling levers at random, PTAL.

**Comment by arielshaqed** [comment]
Here's a sample [result](https://coveralls.io/builds/27834557/source?filename=src/index.ts) for your perusal.

File: package.json [hunk 1]
```diff
@@ -12,7 +12,19 @@
   "scripts": {
     "build": "tsc",
     "prepare": "npm run build",
-    "test": "ava dist/test/**.js"
+    "test": "ava dist/test/**.js",
+    "coverage": "nyc ava dist/test/**.js --reporter=text-lcov"
```
Comment (vogre) [inline package.json]: Looks like this reports avout 0% coverage on `.eslintrc.js` and generates a `coverage/` directory, perhaps needs `-x .eslintrc.js -x coverage`.

**Comment by arielshaqed** [comment]
Thanks!
Pulling in the hope that `master` is somehow magic and the UI will start working for everyone, not just me.

**Comment by arielshaqed** [comment]
Pulling by force: those 3 remaining "build"s are just the old names for "test".  They _should_ go away.

**Pull request merged**